conformance-checking/src/main/java/org/archcnl/conformancechecking/impl/ConformanceCheckOntologyClassesAndProperties.java

```diff
@@ -98,4 +64,59 @@ public String getUri() {
                     + this.name();
         }
     }
+
+    public static Individual createIndividual(ConformanceCheckOntClasses clazz, OntModel model) {
+        return clazz.createIndividual(model);
+    }
+
```

Comment:
You should bring the two createIndivdual() methods up in the file so that the three enums mark the end of the file. This would ensure a better class structure.